### PR TITLE
Improving features app

### PR DESCRIPTION
**Pull request by tjmelo**
Proposal changes 📂 
--
- Adding typescript structure
- Improving performance components
- Implementing error control axios
- Instaling sass resource
- Some expressives enhacements for code

🔥

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,40 @@
...
 
-Initialize project
+Basic Initialize project
 
 ```javascript
-$ npm start
+npm start
 ```
 
+For more instructions about developer envoirment, look at: [package.json](https://github.com/tjmelo/tmx-earth/blob/main/package.json)
```
Comment (tm-wd) [inline README.md:38]:
```suggestion
For more instructions about developer environment, look at: [package.json](https://github.com/tjmelo/tmx-earth/blob/main/package.json)
```

File: src/components/Countries/InfoCountries.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            }).catch(e => console.error(e));
+        }
+    }, [country])
+
+    return(
+        <section className="container">
+            { info }
+        </section>
+    )
+}
```
Comment (tm-wd) [inline src/components/Countries/InfoCountries.tsx:32]:
```suggestion
}

```

File: src/components/Load/index.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { Loading } from "./loading";
+
+export default Loading
```
Comment (tm-wd) [inline src/components/Load/index.ts:3]:
```suggestion
export default Loading

```

File: src/components/Load/loading.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+interface CLoading {
+    type: string
+}
+
+export const Loading:React.FC<CLoading> = ({type, children}) => {
+    return(
+        <div className={`alert alert-${type}`}>{children}</div>
+    )
+}
```
Comment (tm-wd) [inline src/components/Load/loading.tsx:9]:
```suggestion
}

```

File: src/components/Mount/section-list.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+            { data }
+            { subdata 
+            ?  <sup> { subdata } </sup>
+            : '' }
+            </strong>
+            <br />
+        </div>
+
+    )
+}
```
Comment (tm-wd) [inline src/components/Mount/section-list.tsx:25]:
```suggestion
}

```

File: src/model/api.ts [hunk 6]
```diff
@@ -6,4 +6,7 @@ export const api = axios.create({
 })
 
 // Export exclusive list of the name coutries...
-export const apiListCoutries = e => api.get('https://restcountries.com/v3.1/all?fields=name') 
\ No newline at end of file
+export const apiListCoutries = () => 
+    api
+    .get('https://restcountries.com/v3.1/all?fields=name')
+    .catch(error => error.response)
```
Comment (tm-wd) [inline src/model/api.ts:12]:
```suggestion
    .catch(error => error.response)
    
```

File: src/utils/index.ts [hunk 7]
```diff
@@ -0,0 +1,4 @@
+import { parseNumber } from "./parse-number";
+export {
+    parseNumber
+}
```
Comment (tm-wd) [inline src/utils/index.ts:4]:
```suggestion
}

```

**Review by tm-wd — APPROVED** [review]

**Review by tjmelo — COMMENTED** [review]
👍🏼

**Review by tm-wd — APPROVED** [review]

**Review by tm-wd — APPROVED** [review]

**Pull request merged**